File: java/Perro.java
public class Perro {
    private String nombre;
    private String raza;
    private int edad;

    public Perro(String nombre, String raza, int edad) {
        this.nombre = nombre;
        this.raza = raza;
        this.edad = edad;
    }

    // Getters
    public String getNombre() { return nombre; }
    public String getRaza() { return raza; }
    public int getEdad() { return edad; }

    // Setters
    public void setNombre(String nombre) { this.nombre = nombre; }
    public void setRaza(String raza) { this.raza = raza; }
    public void setEdad(int edad) { this.edad = edad; }

    public void ladrar() {
        System.out.println(nombre + " está ladrando.");
    }

    public void jugar() {
        System.out.println(nombre + " quiere jugar.");
    }

    public void comer() {
        System.out.println(nombre + " está comiendo.");
    }

    public static void main(String[] args) {
        Perro miPerro = new Perro("Rex", "Labrador", 3);
        miPerro.ladrar();
        miPerro.jugar();
        miPerro.comer();
    }
}

File: python/perro.py
class Perro:
    def __init__(self, nombre: str, raza: str, edad: int):
        self.__nombre = nombre
        self.__raza = raza
        self.__edad = edad
    
    def ladrar(self) -> None:
        print(f"{self.__nombre} está ladrando: ¡Guau guau!")
    
    def jugar(self) -> None:
        print(f"{self.__nombre} quiere jugar y está saltando alegremente")
    
    def comer(self) -> None:
        print(f"{self.__nombre} está comiendo sabroso")
    
    def get_nombre(self) -> str:
        return self.__nombre
    
    def get_raza(self) -> str:
        return self.__raza
    
    def get_edad(self) -> int:
        return self.__edad

if __name__ == "__main__":
    # Creo un objeto de la clase Perro
    perro1 = Perro("Rex", "Pastor Alemán", 3)
    
    print(f"Información del perro:")  # noqa: F541
    print(f"Nombre: {perro1.get_nombre()}")
    print(f"Raza: {perro1.get_raza()}")
    print(f"Edad: {perro1.get_edad()} años\n")
    
    # Usando los métodos
    perro1.ladrar()
    perro1.jugar()
    perro1.comer()